code/counter.py

```diff
@@ -82,35 +102,106 @@
 			if d in vv:
 				vv.remove(d)
 
-	report.append("### Issue %s (%s)" % (issue.number, issue.title))
-	report.append("  +1: %s [%s]" % (len(votes['+1']), ' '.join(sorted(votes['+1']))))
-	report.append("   0: %s [%s]" % (len(votes['0']), ' '.join(sorted(votes['0']))))
-	report.append("  -1: %s [%s]" % (len(votes['-1']), ' '.join(sorted(votes['-1']))))
+	issue_report = []
+	issue_report.append("### Issue %s (%s)" % (issue.number, issue.title))
+	issue_report.append("  +1: %s [%s]" % (len(votes['+1']), ' '.join(sorted(votes['+1']))))
+	issue_report.append("   0: %s [%s]" % (len(votes['0']), ' '.join(sorted(votes['0']))))
+	issue_report.append("  -1: %s [%s]" % (len(votes['-1']), ' '.join(sorted(votes['-1']))))
+	issue_report.append("")
+
+
+	against = len(votes['0']) + len(votes['-1'])
+	favor = len(votes['+1'])
+	issue_report.append("### Result: %s / %s = %0.2f" % (favor, against+favor, float(favor) / (against+favor)))
+	if float(favor) / (against + favor) > 0.65:
+		issue_report.append("Super majority is in favor, issue is approved")
+		tag = "Approved"
+	elif float(favor) / (against + favor) > 0.49:
```

Comment:
Shouldn't this be `>= 0.5` or '>0.4999` rather than '>0.49'? I don't see a 1% error from floating point math

Or if you want to do perfect integer math at the expense of less readable code, `favor >= against` is actually the same ;-)